.github/workflows/PR-check.yml

```diff
@@ -0,0 +1,51 @@
+---
+name: PR Check
+
+on: 
+#  pull_request:
+#    paths-ignore:
+#      - '.github/workflows/**'
+#      - 'test-script'  
+  workflow_dispatch:
+
+env:
+  ARTIFACT_VERSION: dummy-verison 
+
+jobs:
+  build:
+    name: PR Check
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      
+      - name: Maven Cache 
+        uses: actions/cache@v3
+        with:
+          key: cache-${{ github.workflow }}
+          path: |
+            ~/.m2/repository
+
+      - uses: ./.github/actions/GCPSetup
+        with:
+          GCPCreds: ${{ secrets.GCP_SA_JSON }} 
+
+      - name: 'Maven settings'
+        run: 'gsutil cp gs://salus-mavenrepository/m2-settings.xml ./.mvn/settings.xml'
+
+      - name: .m2 forder [*** fix it please ***]
+        run: sed -i 's*/root/.m2/*~/.m2/*g' ./.mvn/settings.xml
+
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: '14'
+          distribution: 'adopt-hotspot'
+
+      - name: Build with Maven
+        shell: bash
+        run: |
+          mvn -B verify -s .mvn/settings.xml 
+
+      - name: Build with jib
+        run: mvn compile jib:dockerBuild -U -B -Dmaven.test.skip=true -Ddocker.image.prefix=dummy    
```

Comment:
Why wouldn't you just add the docker build to the previous `mvn` command? Running maven again will just be unnecessary overhead.
```
mvn -B -s .mvn/settings.xml verify jib:dockerBuild -Ddocker.image.prefix=dummy
```